scopes.cwt

```diff
@@ -1900,4 +1900,9 @@ alias[effect:random_owned_bloodline] = {
 		alias_name[trigger]=alias_match_left[trigger]
 	}
 	alias_name[effect] = alias_match_left[effect]
-}
\ No newline at end of file
+}
+
+## push_scope = character
+## scope = character
+### 
+alias[trigger:faction_claimant] = single_alias_right[trigger_clause]
```

Comment:
Is this a "link", e.g. a scope change that works anywhere?
If yes, move to links.cwt

Reply:
Afaik it is so it's been moved.